src/enzyme_ad/jax/Passes/StablehloOptPatterns.td

```diff
@@ -0,0 +1,61 @@
+//===- Passes.td - EnzymeXLA pass tablegen macros  ------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+include "mlir/IR/OpBase.td"
+include "stablehlo/dialect/StablehloOps.td"
+
+////////
+// Mul/Div || Add/Sub - constant computation lifting 
```

Comment:
We can probably remove the ones added in https://github.com/EnzymeAD/Enzyme-JAX/pull/301 in favor of these ones

Reply:
done, merged the binop_const tests into the new test file